dev/requirements.txt

```diff
@@ -59,3 +59,7 @@ googleapis-common-protos==1.56.4
 mypy-protobuf==3.3.0
 googleapis-common-protos-stubs==2.2.0
 grpc-stubs==1.24.11
+
+# TorchDistributor dependencies
+torch==1.13.1
+torchvision==0.14.1
```

Comment:
You actually don't need to pin this version. This is only for developement enviornment, and I would prefer to let other people test it with the latest versions.

Reply:
+1 for @HyukjinKwon 's comment. And, this should be mentioned as test dependency explicitly.